docs/ug/cli.md

```diff
@@ -110,7 +110,7 @@ The last modified dates will be in the same timezone specified with the `--timez
 ### `--output`, `-o`
 
 **`--output OUTPUT_DIRECTORY`**: Indicates where to save the report generated.
-* Parameter: `OUTPUT_DIRECTORY` location for the generated `reposense-report` folder<br>
+* Parameter: `OUTPUT_DIRECTORY` The location for the generated `reposense-report` folder.<br>
```

Comment:
Above this is `last-modified-date`. Can add a full stop after "authorship.json".